tools/crm_simulate.c

```diff
@@ -306,15 +306,18 @@ create_action_name(pe_action_t *action)
     return action_name;
 }
 
-static void
-create_dotfile(pe_working_set_t * data_set, const char *dot_file, gboolean all_actions)
+static bool
+create_dotfile(pe_working_set_t * data_set, const char *dot_file, gboolean all_actions,
+               GError **error)
 {
     GListPtr gIter = NULL;
     FILE *dot_strm = fopen(dot_file, "w");
 
     if (dot_strm == NULL) {
-        crm_perror(LOG_ERR, "Could not open %s for writing", dot_file);
-        return;
+        g_set_error(error, G_OPTION_ERROR, pcmk_rc2exitc(errno),
+                    "Could not open %s for writing: %s", dot_file,
+                    pcmk_rc_name(errno));
```

Comment:
I think you want pcmk_rc_str(errno) here, pcmk_rc_name will just print EWHATEVER

Reply:
Done.